src/macros.rs

```diff
@@ -1,3 +1,4 @@
+use std::env;
```

Comment:
This is only required for the test, right?

Reply:
Correct. I should scope this import.